modules/accumulo/src/test/java/org/apache/fluo/accumulo/iterators/TimeSkippingIteratorTest.java

```diff
@@ -25,7 +25,7 @@
 import org.apache.accumulo.core.data.Range;
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
-import org.apache.accumulo.core.iterators.SortedMapIterator;
+import org.apache.accumulo.core.iteratorsImpl.system.SortedMapIterator;
```

Comment:
Isn't there a public API/SPI equivalent available?

Reply:
I did some grepping around and I didn't find anything but this.   I am not sure why this class moved in Accumulo, I didn't look into that.  Using the impl class here in Fluo seems ok because its Fluo test code.

Reply:
I think this moved as part of the goal to make the user iterators public API.

Reply:
Long-term, probably best to use the iterator test harness, but for now, it's probably not a problem to rely on this non-public API for tests.